config/user-commands.m4

```diff
@@ -26,10 +26,12 @@ AC_DEFUN([ZFS_AC_CONFIG_USER_COMMANDS_COMMON], [
 	AC_PATH_TOOL(DU, du, "")
 	AC_PATH_TOOL(ECHO, echo, "")
 	AC_PATH_TOOL(EGREP, egrep, "")
+	AC_PATH_TOOL(FALSE, false, "")
 	AC_PATH_TOOL(FDISK, fdisk, "")
 	AC_PATH_TOOL(FGREP, fgrep, "")
 	AC_PATH_TOOL(FILE, file, "")
 	AC_PATH_TOOL(FIND, find, "")
+	AC_PATH_TOOL(FIO, fio, "/usr/bin/fio")
```

Comment:
You shouldn't need to specify "/usr/bin/fio", this is a standard package available for most distributions.